includes/class-wc-cache-helper.php

```diff
@@ -94,7 +94,7 @@ public static function get_cache_prefix( $group ) {
 		$prefix = wp_cache_get( 'wc_' . $group . '_cache_prefix', $group );
 
 		if ( false === $prefix ) {
-			$prefix = 1;
+			$prefix = microtime();
```

Comment:
You will need to pass in true otherwise it will return a string which contains spaces which in turn will break the key

Reply:
I based the implementation on the discussion in the core ticket (https://core.trac.wordpress.org/ticket/23448) and it seemed to work fine in the test, so marking as resolved. 

Thanks for the review!